_includes/manuals/nightly/3.3.1_rpm_packages.md

```diff
@@ -12,7 +12,7 @@ All RHEL and derivatives must be [subscribed to EPEL](https://fedoraproject.org/
 #### Pre-requisites: Software collections
 All RHEL and derivatives require software collections from the CentOS SCLorg Special Interest Group (SIG). Foreman uses the `rh-ruby25` collection - which is from RHSCL.
 
-Generally we recommend all users to use the CentOS SCLorg SIG's builds, which can be configured by installing the `foreman-release-scl` package from our repositories (which in turn installs the `centos-release-scl-rh` package). The Foreman installer will use this package by default.
+Generally we recommend all users to use the CentOS SCLorg SIG's builds, which can be configured by installing the `centos-release-scl-rh` package. The Foreman installer will use this package by default.
```

Comment:
The last part is not true anymore. While it does that by default (even on RHEL, which might be its own issue), we now require `centos-release-scl-rh` to be present before installation due to our SELinux hacks.